Appraise/urls.py

```diff
@@ -192,6 +192,11 @@
         campaign_views.campaign_status,
         name='campaign_status',
     ),
+    re_path(
+        r'^pairwise-assessment-document-esa/$',
+        evalview_views.pairwise_assessment_document_esa,
+        name="pairwise-assessment-document-newui-esa",

```

Comment:
I'd suggest to make the URL shorter:
```suggestion
        name="pairwise-assessment-document-esa",

```

Reply:
Changed.